TrackForce/src/main/webapp/js/mainApp.js

```diff
@@ -246,7 +261,38 @@ mainApp.controller("clientCtrl", function($scope, $http) {
 									clients.selectedUnmapped,
 									clients.confirmedUnmapped,
 									clients.deployedUnmapped ] ];
-					$scope.options = { legend: { display: true, position: 'right' } };
+					$scope.options = {
+						legend : {
+							display : true,
+							position : 'right'
+						}
+					};
 				});
 	}
 });
+// controller for database population and deletion
+mainApp
+		.controller(
+				'databaseCtrl',
+				function($http, $scope) {
+					$scope.populateDB = function() {
+						$http(
+								{
+									method : "GET",
+									url : "http://localhost:8080/TrackForce/track/database/populateDB"
+								}).then(function(response) {
+							$scope.dbMessage = response.data;
+
+						})
+					}
+					$scope.deleteDB = function() {
+						$http(
+								{
+									method : "GET",
+									url : "http://localhost:8080/TrackForce/track/database/deleteFromDB"
+								}).then(function(response) {
+							$scope.dbMessage = response.data;
+
+						})
+					}
+				});
```

Comment:
The formatting of this code looks extreme. Can it be reformatted to be paired down?